Issue#10 add alt attributes to images

amadulhaxxani opened the pull request:
## Fixed
- Added appropriate alt attributes to img tags
- Ensured all icons have meaningful descriptions for accessibility compliance

amadulhaxxani committed:
adding alt in the img tags

adding alt in the img tags

amadulhaxxani committed:
fixed the alt texts for icon

fixed the alt texts for icons from WCAG 1.1.1 Non-text Content #33 issue

amadulhaxxani committed:
fixed linting errors

fixed linting errors

kuchtiak-ufal commented on src/assets/i18n/en.json5:
```diff
@@ -5984,4 +5984,30 @@
 
   "clarin-license-agreement-page.download-error": "An error occurred while downloading the file. Please try again later.",
+
+  "home.image.glass": "Home page glass maginier image",
```
is "glass maginier" correct expression

amadulhaxxani replied:
I will fix it

kosarko commented on src/assets/i18n/cs.json5:
```diff
@@ -8435,4 +8435,40 @@
   // "clarin-license-agreement-page.download-error": "An error occurred while downloading the file. Please try again later.",
   "clarin-license-agreement-page.download-error": "Při stahování souboru došlo k chybě. Zkuste to prosím znovu později.",
+  // "home.image.glass":"Home page glass maginier image",
+  "home.image.glass": "Obrázek skleněného lupa na domovské stránce",
```
either @kosarko or @kuchtiak-ufal will fix the czech translations

amadulhaxxani committed:
fixed the typo maginier - magnifier

fixed the typo maginier - magnifier

kosarko committed:
updated czech translations

amadulhaxxani committed:
added test for formateIconsAltText function

added test for formateIconsAltText function and a new case for a string with underscore

amadulhaxxani committed:
Merge branch 'Issue#10-Add-alt-attributes-to-images' of https://github.com/ufal/dspace-angular into Issue#10-Add-alt-attributes-to-images

kosarko committed:
reverting unnecessary formating

kosarko committed:
reverting formating changes

kosarko committed:
reverting formating changes
